extensions/mutiny/deployment/src/main/java/io/quarkus/mutiny/deployment/MutinyProcessor.java

```diff
@@ -23,4 +23,16 @@ public void initExecutor(ExecutorBuildItem executorBuildItem, MutinyInfrastructu
         ExecutorService executor = executorBuildItem.getExecutorProxy();
         recorder.configureMutinyInfrastructure(executor);
     }
+
+    @BuildStep
+    @Record(ExecutionTime.STATIC_INIT)
+    public void defineDroppedExceptionHandler(MutinyInfrastructure recorder) {
+        recorder.configureDroppedExceptionHandler();
```

Comment:
[Picky] - We could have done everything in a single method.

Reply:
Ok let me change that. I was wondering if that was a pattern to delegate to a class there, but since the actual code is in runtime and we're in a static init... yeah 👍

Reply:
Nevermind I get it now